apps/coa/views/partials/summary-table.html

```diff
@@ -0,0 +1,28 @@
+<div>
+  <h2 class="section-header">{{section}}</h2>
+</div>
+<dl class="section-fields">
+  {{#fields}}
+  {{^omitFromSummary}}
+    <div class="confirm-row">
+      {{#isSeparator}}
+      {{/isSeparator}}
+      {{^isSeparator}}
+        <dt class="confirm-label">{{label}}</dt>
+        <dd class="confirm-value" data-value="{{value}}">{{value}}</dd>
+      {{^omitChangeLink}}
+        {{#changeLink}}
+          <dd><span><a class="govuk-link" href="{{changeLink}}" id="{{field}}-change-{{index}}">{{#t}}buttons.change{{/t}} <span class="visuallyhidden">{{changeLinkDescription}} from {{value}}</span></a></span></dd>
```

Comment:
couldn't see a translation for `changeLinkDescription`, is there a default value?

Reply:
The `changelinkdescription` is defined in hof/components/summary/index.js dynamically like below
 The value from the translation for summary sections label is assigned to the variable `changeLinkDescription`
` changeLinkDescription: this.translateChangeLink(inner.field, req),`
```
  translateChangeLink(key, req) {
    return req.translate([fields.${key}.changeLinkDescription,
      pages.confirm.fields.${key}.label,
      fields.${key}.summary,
      fields.${key}.label,
      fields.${key}.legend]);
  }
```